apps/codeforafrica/src/utils/camelcaseKeys.js

```diff
@@ -0,0 +1,11 @@
+import camelcaseKeys from "camelcase-keys";
+
+export default function convertToCamelCase(obj) {
+  if (Array.isArray(obj)) {
+    return obj.map((item) => convertToCamelCase(item, { deep: true }));
+  }
+  if (typeof obj === "object" && obj !== null) {
+    return camelcaseKeys(obj, { deep: true });
+  }
+  return obj;
+}
```

Comment:
We don't need this function. `camelcase-keys` should be able to handle objects and arrays natively.